exp/runtime/hooks/api/v1alpha1/discovery_types.go

```diff
@@ -44,33 +44,40 @@ type DiscoveryResponse struct {
 	// handlers defines the current ExtensionHandlers supported by an Extension.
 	// +listType=map
 	// +listMapKey=name
+	// +required
```

Comment:
```suggestion
	// +optional
```
\+ let's add omitempty

Similar to the last finding. Not part of a CRD + this field is actually optional